### PR TITLE
allow julia 1.0

### PR DESCRIPTION
**Pull request by oxinabox**
Follow up to #20

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -10,7 +10,7 @@ struct Foo end
 # if code has no calls then it must be fully static
 has_no_calls(ir) = all(stmt->!Meta.isexpr(stmt, :call), ir)
 
-@testset "static_hasmethod" begin
+VERSION >= v"1.3" && @testset "static_hasmethod" begin
```
Comment (aminya) [inline test/runtests.jl:13]: I think you should use normal if here 🤔 or use paranteces around `@testset`
```suggestion
if VERSION >= v"1.3" 
	@testset "static_hasmethod" begin
```

Comment (oxinabox) [reply test/runtests.jl:13]: I disagree, the extra indent adds nothing and this is a common enough pattern for testsets.

File: test/runtests.jl [hunk 2]
```diff
@@ -81,7 +81,7 @@ h(::Int) = 1
 end
 using .Bar
 
-@testset "static_methods" begin
+VERSION >= v"1.3" @testset "static_methods" begin
```
Comment (oxinabox) [inline test/runtests.jl:84]:
```suggestion
VERSION >= v"1.3" && @testset "static_methods" begin
```

File: test/runtests.jl (hunk 1 above)
Comment (aminya) [reply test/runtests.jl:13]: I thought the parser has some issues parsing this. Turns out it was another sentence.

**Pull request merged**